packages/@glimmer/integration-tests/test/modifiers-test.ts

```diff
@@ -82,16 +82,27 @@ class ModifierTests extends RenderTest {
   }
 
   @test
-  'do not work on component invocations'(assert: Assert) {
-    this.registerComponent('Glimmer', 'Foo', '<div ...attributes>Foo</div>');
-    this.registerModifier('bar', BaseModifier);
-    assert.throws(() => {
-      this.render('<Foo {{bar foo="foo"}} />');
-    }, 'Compile Error: Element modifiers are not allowed in components');
-
-    assert.throws(() => {
-      this.render('<Foo (bar foo="foo") />');
-    }, 'Compile Error: Element modifiers are not allowed in components');
+  'work on component invocations'(assert: Assert) {
+    let modifierParams = null;
+    let modifierNamedArgs = null;
+    let modifiedElement: SimpleElement | undefined;
+    class Bar extends AbstractInsertable {
+      didInsertElement(params: Dict<unknown>, namedArgs: Dict<string>) {
```

Comment:
sorry, I should be more clear, I meant `params: unknown[], namedArgs: Dict<unknown>`